custom_components/elasticsearch/es_index_manager.py

```diff
@@ -113,6 +110,23 @@ async def _create_index_template(self):
         else:
             LOGGER.debug("Creating index template")
 
+        # For Datastream mode we do not offer configuration wtihin Home Assistant for the ILM policy
+        if not self._gateway.es_version.meets_minimum_version(major=8, minor=11):
+            LOGGER.debug("Running pre-8.10, using Index Lifecycle Management")
+            await self._create_basic_ilm_policy(
+                ilm_policy_name=DATASTREAM_METRICS_ILM_POLICY_NAME
+            )
+
+            LOGGER.debug("Inserting ILM Policy into Index Template: ")
+
+            del index_template["template"]["lifecycle"]
+
+            index_template["template"]["settings"]["index.lifecycle.name"] = (
+                DATASTREAM_METRICS_ILM_POLICY_NAME
+            )
```

Comment:
**question** are these change the delta between what ILM supports/expects, and what DLM supports/expects?